plugins/@grouparoo/bigquery/__tests__/table-import/import-property.ts

```diff
@@ -189,31 +189,31 @@ describe("bigquery/table/recordProperty", () => {
           sourceMapping,
           aggregationMethod: "average",
         });
-        expect(value).toEqual([1.73]);
+        expect(value).toEqual([1.6257142857142857]);
```

Comment:
If you are having trouble with rounding/avg aggregation numbers like this in tests, we sometimes use a "float helper"

https://github.com/grouparoo/grouparoo/blob/fbc78c3ba43fd1974a5c6bdff6cabb33a78126c7/plugins/%40grouparoo/mysql/__tests__/table-import/import-properties.ts#L1151-L1153